pyproject.toml

```diff
@@ -269,7 +269,6 @@ extend-select = [
   "RUF013", # implicit-optional
   "RUF015", # unnecessary-iterable-allocation-for-first-element
   "RUF016", # invalid-index-type
-  "RUF018", # assignment-in-assert
```

Comment:
Why remove this rule?

Reply:
This is an experimental rule recently added by ruff, and I was locally getting warning messages about not running ruff with ``--preview`` enabled. It's unrelated to this PR.